src/main/java/org/ballerinalang/command/util/ToolUtil.java

```diff
@@ -783,6 +783,20 @@ private static String getServerURL() {
      * @param version distribution version
      */
     public static String getType(String version) {
-        return version.contains("1.") ? "jballerina" : "ballerina";
+        if (version.contains("1.")) {
```

Comment:
@parkavi11 This function should specify only its ballerna (which is swanlake) or jballerina. We should not include Preview parts here